.github/workflows/create_release.yml

```diff
@@ -0,0 +1,51 @@
+name: Create Release
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        required: true
+        type: string
+
+jobs:
+  release:
+    permissions:
+      # Necessary permissions to create a release.
+      contents: write
+    env:
+      BUILD_DIR: build
+      RELEASE_BIN_ARCHIVE: release_bin.tar.gz
+      RELEASE_SRC_ARCHIVE: release_src.tar.gz
+      DEFAULT_BRANCH: refs/heads/${{ github.event.repository.default_branch }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
Pleasantly surprised that Qemu builds in ~5 minutes :)

https://github.com/pamaury/qemu-ot/actions/runs/5826505885/job/15800622550

Reply:
It builds in 1 minute on a laptop :-)
```
time (mkdir tmpbuild && cd tmpbuild && ../configure --target-list=riscv32-softmmu --without-default-features --enable-tcg --enable-tools --enable-trace-backends=log && ninja)
202.48s user 60.76s system 440% cpu 59.739 total
```

Reply:
Wow it definitely does not build in one minute on my laptop :-o Maybe that's the GTK stuff?

Reply:
Apple's cores are quite good for build tasks (M1 pro in this case).

Reply:
I have removed the GTK stuff but it only removes a few seconds, still takes about 5 minutes to build it with GitHub actions, not too bad though.